docs/UserGuide.md

```diff
@@ -182,13 +209,12 @@ Examples:
 
 Shows a list of all students of a specified class.
 
-Format:
+Format:`list_student INDEX [tn/TUTORIAL_NAME]`
```

Comment:
I think this should be changed to `list_student INDEX` or `list_student tn/TUTORIAL_NAME`